src/scenes/gameuiscreen.ts

```diff
@@ -1,28 +1,51 @@
 import { ScreenHeight, ScreenWidth } from "core/main";
 import Phaser from "phaser";
 import { log } from "utilities/logger";
-import { interactify } from "utilities/utils";
+import { assert, interactify } from "utilities/utils";
+import { float, int, Nullable } from "../definitions/utils";
+import { Tile } from "../entities/Tile";
+import TileManager from "../managers/TileManager";
 
 export class GameUiScreen extends Phaser.Scene {
+    private back: Nullable<Phaser.GameObjects.Rectangle>;
+
     public constructor() {
         super("GameUiScreen");
+        this.back = null;
+    }
+
+    public preload(): void {
+        for (let i: int = 0; i < 9; i++) {
+            this.load.image(`tile${i}`, `assets/tiles/tile${i}.png`);
+        }
     }
 
     public create(): void {
-        const back: Phaser.GameObjects.Rectangle = this.add.rectangle(
+        this.back = this.add.rectangle(
             ScreenWidth / 2,
             ScreenHeight - 100,
             ScreenWidth * 0.8,
             200,
-            0xffffff,
+            0x808080,
         );
-        back.setInteractive();
+        this.back.setInteractive();
+        this.displayDrawnTiles();
```

Comment:
The problem that i think @paul891112 also mentioned is that the tiles only get created once at the beginning but its better to make it update/event based... like this:
- make a container (which holdes all tiles in the bar)
- move the display function into the tilemanager sync listener
- at the beginning of the display function all the old tiles from the container get removed
- all the new ones get created based on the tilemanager getInHand and put into the container

... this way each time the tilemanager fires a event that the hand changed all the old tiles get destroyed and replaced with the new accurate ones... to see how this is done look at @paul891112 s PR with the world tiles.